src/components/explore/ProbeExplorer.svelte

```diff
@@ -215,31 +233,53 @@
 </div>
 
 <div class="graphic-and-summary" class:no-line-chart={justOne}>
-  <div style="display: {justOne ? 'none' : 'block'}">
-    <AggregationsOverTimeGraph
-      title={aggregationsOverTimeTitle}
-      description={aggregationsOverTimeDescription}
-      {data}
-      xDomain={$domain}
-      {yDomain}
-      lineColorMap={binColorMap}
-      {key}
-      yAccessor={overTimePointMetricType}
-      xScaleType={aggregationLevel === 'version' ? 'scalePoint' : 'time'}
-      {yScaleType}
-      {yTickFormatter}
-      metricKeys={activeBins}
-      {ref}
-      {hovered}
-      bind:hoverValue
-      {aggregationLevel}
-      on:click={() => {
-        if (hovered.datum) {
-          ref = hovered.datum;
-        }
-      }}>
-      <slot name="additional-plot-elements" />
-    </AggregationsOverTimeGraph>
+  <div style="display: block">
+    {#if justOne}
+      <div class="data-error-msg">
+        <div class="data-error-msg__bg">
+          <GlamErrorShapes />
+        </div>
+        <div>
+          <p class="detail-title">
+            Currently we don't have enough data to generate an over-time graph
+            for this metric. Please refer to the Table View to see the available
+            data, or use the SQL query generator for <a
+              href="https://docs.telemetry.mozilla.org/cookbooks/main_ping_exponential_histograms.html"
+              >further STMO analysis</a
+            >.
+            <br /> <br />
```

Comment:
I think you can achieve the desired spacing between elements with the `detail-title` css class instead of using `<br />` tags

Reply:
thank you both! this is now fixed!